{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/cli.py

```diff
@@ -6,136 +6,159 @@
 render model specs with the correct location parameters plugged in.
 
 It will look for the model spec template in "model_spec.in" in the directory
-"../{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/model_specification".
-Specify multiple locations in a file called "locations.txt" in the same directory.
-You can create a single location using the "-s" option. Using the single
-location option will override an existing locations file.
+``src/{{cookiecutter.package_name}}/model_specifications``.
+Specify multiple locations in a file called "locations.txt" in the same
+directory. You can create a single location using the "-s" option. Using the
+single location option will override an existing locations file.
 
 The application will look for the model spec and locations files based
 on the python environment that is active and these files don't need
 to be specified if the default names and location are used.
 """
+from collections import namedtuple
+from pathlib import Path
 import re
+from typing import List, Optional, Generator
+
 import click
-from pathlib import Path
 from jinja2 import Template
-from collections import namedtuple
 from loguru import logger
-from typing import List, Callable
-from functools import wraps
-from sys import exit
 
 
-PROJ_NAME = '{{cookiecutter.package_name}}'
-Location = namedtuple('Location', ['loc_proper', 'loc_sanitized'])
+PROJECT_NAME = '{{cookiecutter.package_name}}'
+Location = namedtuple('Location', ['proper', 'sanitized'])
+
+
+def sanitize(*locations: str) -> Generator[Location]:
+    """Processes locations into tuples of proper and sanitized names.
+
+    Sanitized location strings are all lower case, have spaces replaced
+    by underscores, and have apostrophes replaced by dashes.
+
+    Parameters
+    ----------
+    locations
+        The locations to process formatted as proper location names.
+        Proper location names should come from GBD location set 1, the
+        location reporting hierarchy wherever possible. If using sub-national
+        locations, they may be found in GBD location set 35, the model
+        results location hierarchy.
+
+    Yields
+    -------
+        Named tuples with both the proper and sanitized location names
+
+    Examples
+    --------
```

Comment:
I don't typically write examples sections, but since I don't know how to back this repository with tests, it seemed useful here.